charts/telepresence/templates/deployment.yaml

```diff
@@ -94,6 +94,8 @@ spec:
             containerPort: 8081
           - name: https
             containerPort: 8443
+          - name: prometheus
+            containerPort: 8082
```

Comment:
Can we please avoid having opinions about default ports? If users want prometheus, they should decide the port themselves. Perhaps just require this port to be configured in order to enable prometheus? That's consistent with how we enable the API server.

Reply:
altho i agree that prometheus should be disabled by default and that the port should be configurable we should still provide a default port; something like:


```
prometheus:
  enabled: false
  port: 8082
```

Just so a sysadmin can go `helm install ... --set prometheus.enabled=true` and have it magically spin everything up without worrying about ports. I say this bc if I put on my sysadmin hat the truth is I don't necessarily know or care which ports the app I'm installing uses already (i.e. I don't know if my choice will cause a conflict) and I'm just trying to enable prometheus and move on with my day

Reply:
@josecv why should we provide a default port? I just see problems on the horizon. People doing `prometheus.enabled=true` and running into port conflicts, not knowing what's causing it. If they instead must to `prometheus.port=8082`, then they cause the conflict themselves.

What is the motivation for using port `8082`? Is that normally used for prometheus traffic?